aspnetcore/host-and-deploy/azure-apps/troubleshoot.md

```diff
@@ -70,12 +70,12 @@ An alternative to using the **Diagnose and solve problems** blade is to examine
 
 ### Run the app in the Kudu console
 
-Many startup errors don't produce useful information in the Application Event Log. Run the app in the Kudu Remote Execution Console to discover the error:
+Many startup errors don't produce useful information in the Application Event Log. You can run the app in the Kudu Remote Execution Console to discover the error:
```

Comment:
What??!! "You" ... "You" ... I can't believe *you* put a "You" in here. :smile: lol

Reply:
Use you when it make it more clear. In this case you makes it more clear. Avoid you when it doesn't add value.

Reply:
I see ... sooooo ...

**YOU is not banned from the repo!** :trollface: lol

... you should'a known that was going to come back to haunt u. 👻 😝